SU-SWS/stanford_basic
# D8CORE-3502 - a11y fixes for nav menu

# READY FOR REVIEW

# Summary
- This PR addresses a number of a11y concerns related to the main nav menu.

# Review By (Date)
- 5/6/2021

# Urgency
- medium

# Steps to Test

1. Checkout this branch.  Clear your caches, including browser cache.
2. Enable drop-down menus, create a bunch of menu items, and nest them however you want.
3. View the nav menu in the front end. verify:
	- `aria-expanded` no longer appears on `<a>` tags, but DOES appear on the drop-down toggles.
	- the `aria-label` on the drop down toggles no longer says 'expand menu', but rather mentions the name of the menu the toggle opens.
	- the `esc` key returns focus to the drop-down toggle when used to close a menu, rather than to the parent menu item.
	- focus indicators of both color and underline are present on the drop-down toggles in both firefox and edge, if possible.  I tested this myself, and I found they were present, so I didn't change anything about them, but it was noted as a bug in the ticket.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
